mcs_benchmark_data/models/benchmark_dataset.py

```diff
@@ -18,14 +18,13 @@ class BenchmarkDataset(_Model):
     entries: Tuple[BenchmarkSample,...]
 
     def to_rdf(
-        self, *, graph: Graph, **kwds) -> Resource:
+        self, *, graph: Graph) -> Resource:
         resource = _Model.to_rdf(
-            self, graph=graph, **kwds
+            self, graph=graph
         )
-        resource.add(RDF.type, MCS[self.__class__.__name__])
 
-        #How to add name?
+        resource.add(SCHEMA.name, self.name)
         for entry in self.entries:
-            entry.to_rdf(add_to_resource=resource)
+            entry.to_rdf(graph)

```

Comment:
Have to do graph=graph because of the self, *, graph (* in this context means all parameters must be specified with keywords).